scripts/generate.sh

```diff
@@ -19,6 +19,7 @@ fi
 
 echo "VERSION: $VERSION"
 
-sed -i "s/projectVersion: .*/projectVersion: $VERSION/g" gradle.properties
-sed -i "s/^VERSION=.*/VERSION='$VERSION'/g" ./scripts/generate.sh
+sed -i '' -e "s/projectVersion = .*$/projectVersion = $VERSION/g" ./gradle.properties
```

Comment:
You just broke it for linux, try to check the platform like [here](https://github.com/fingerprintjs/fingerprint-pro-server-api-python-sdk/blob/main/generate.sh#L47)